ej2-react/document-editor/print.md

```diff
@@ -1,16 +1,16 @@
 ---
 layout: post
-title: Print in React Document editor component | Syncfusion
-description: Learn here all about Print in Syncfusion React Document editor component of Syncfusion Essential JS 2 and more.
+title: Print in React Document Editor Component | Syncfusion
+description: Learn here all about Print in Syncfusion Essential React Document Editor component, its elements and more.
 control: Print 
 platform: ej2-react
 documentation: ug
 domainurl: ##DomainURL##
 ---
 
-# Print in React Document editor component
+# Print in React Document Editor Component
 
-To print the document, use the [`print`](https://ej2.syncfusion.com/react/documentation/api/document-editor#print) method from document editor instance.
+To print the document, use the [`print`](https://helpej2.syncfusion.com/react/documentation/api/document-editor/#print) method from document editor instance.
```

Comment:
why you add helpej2 link?
Check and correct links in all places.

Reply:
removed